base/applications/mscutils/eventvwr/eventvwr.c

```diff
@@ -3807,6 +3807,36 @@ WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
             break;
         }
 
+        case WM_CONTEXTMENU:
```

Comment:
Please move this code to where there was an existing `WM_CONTEXTMENU` that was commented out.